### PR TITLE
feat: gravsearch count viewer implemented

### PR DESCRIPTION
**Pull request by EricSommerhalder**
resolves DEV-680

### CODE REVIEW

File: src/GravsearchViewer/GravsearchCountViewer.elm [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+update msg model =
+    case msg of
+        Received (Ok count) ->
+            ( { model | result = Just count }, Cmd.none )
+
+        Received (Err err) ->
+            ( { model | error = Just (buildErrorMessage err) }, Cmd.none )
+
+        Fetch ->
+            ( { model | result = Just { numberOfItems = -2 } }, fetchResult model )
```
Comment (mdelez) [inline src/GravsearchViewer/GravsearchCountViewer.elm:53]: I don't think this "-2" needs to be here

File: src/GravsearchViewer/GravsearchCountViewer.elm [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+view model =
+    div []
+        [ div [] [ text model.query ]
+        , primaryButton [ onClick Fetch ] "Search" BasicLarge
+        , resView model
+        , errorView model.error
+        ]
+
+
+resView : Model -> Html msg
```
Comment (mdelez) [inline src/GravsearchViewer/GravsearchCountViewer.elm:70]: according to our other function names, the name of this function and the function for the error view should start with "view"

File: src/Projects/TailwindPlayground.elm [hunk 3]
```diff
@@ -17,24 +18,28 @@ import Text.Accordion as Accordion
 import Text.ProjectDescription as ProjectDescription
 import Tiles.ImageTile as ImageTile
 import Tiles.ImageTileGrid as ImageTileGrid
+import Tuple exposing (first)
```
Comment (mdelez) [inline src/Projects/TailwindPlayground.elm:21]: this appears to be unused and can be removed

**Review by mdelez — CHANGES_REQUESTED** [review]
Just some clean up to do :)

File: src/GravsearchViewer/GravsearchCountViewer.elm (hunk 2 above)
Comment (EricSommerhalder) [reply src/GravsearchViewer/GravsearchCountViewer.elm]: Completely agree

**Review by mdelez — APPROVED** [review]
works well, thanks!

**Pull request merged**